accounts-tool/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "solana-accounts-tool"
+version = "1.15.0"
+description = "Solana Accounts Tool"
+authors = ["Solana Maintainers <maintainers@solana.foundation>"]
+repository = "https://github.com/solana-labs/solana"
+homepage = "https://solana.com/"
+documentation = "https://docs.rs/solana-accounts-tool"
+license = "Apache-2.0"
+edition = "2021"
+
+[dependencies]
+clap = "2.33.1"
+log = { version = "0.4.17" }
+solana-logger = { path = "../logger", version = "=1.15.0" }
+solana-measure = { path = "../measure", version = "=1.15.0" }
+solana-version = { path = "../version", version = "=1.15.0" }
```

Comment:
Same as comment in other issue, can you please update these to deps to pull versions from the project level Cargo.toml file

Reply:
Rebased, but it seems the PR you mentioned got reverted.

Reply:
Yep, mentioned in the other PR but not here:
https://github.com/solana-labs/solana/pull/29635#discussion_r1086310631